docs/examples/classification.ipynb

```diff
@@ -0,0 +1,946 @@
+      "text/plain": [
+       "<Figure size 800x800 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(8, 8))\n",
+    "plot_bias(\n",
+    "    y_obs=y_test,\n",
+    "    y_pred=df_pred_test,\n",
+    "    feature=X_test[\"parents\"],\n",
+    "    confidence_level=0.8\n",
+    ");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Predictive Model Performance\n",
+    "### 4.1 Score Decomposition of the Gamma Deviance\n",
+    "We use the [log loss](https://lorentzenchr.github.io/model-diagnostics/reference/model_diagnostics/scoring/scoring/#model_diagnostics.scoring.scoring.LogLoss) and evaluate on the test set.\n",
```

Comment:
It would be nice to also report the squares error decomposition. The squared error is called Brier score in the context of probability predictions / classification and the decomposition was originally defined / discovered on the Brier score.

Reply:
At the very bottom of the notebook there should be the decomposition of the squared error

![Bildschirmfoto 2023-10-01 um 19 51 16](https://github.com/lorentzenchr/model-diagnostics/assets/124086916/d7d56549-5ddf-4277-a54e-8d19fa601d6b)